experimental/addons/rabbitmq-operator/README.md

```diff
@@ -0,0 +1,57 @@
+# rabbitmq-operator
+
+This is an addon template. Check how to build your own addon: https://kubevela.net/docs/platform-engineers/addon/intro
```

Comment:
```suggestion

```
You can try to give some introduction to rabbitmq-operator.

Reply:
Great, I saw you add the introduction, but you forgot to remove the placeholder? *This is an addon template...*